index.html

```diff
@@ -11,86 +11,139 @@
 </head>
 
 <body>
-
-	<h1>You got this!  Good luck.</h1>
-
-<!-- Menu Content 
-S&J
-
-Home
-Services
-Contact
--->
-
-
-<!-- Home Page Content:
+<div class="container"> <!-- Container -->
+	<!-- Menu Content -->
+
+	<div class="dropdown">	
+		<div class="nav">
+			<div class="header">
+				<h1>S&J</h1>
```

Comment:
One of these divs is unnecessary. You can apply the classes nav and header to the outermost div if you need, though.